uncontrast_studies/models/experiment.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from django.db.models import CASCADE
+from simple_history.models import HistoricalRecords
+
+from contrast_api.choices import ExperimentTypeChoices
+
+
+class UnConExperiment(models.Model):
+    """
+    This is a major model here, lots of data "belongs" to the experiment in related models, so the experiment is codified there
+    """
+
+    class Meta:
+        ordering = ["id"]
+
+    study = models.ForeignKey(to="studies.Study", on_delete=CASCADE, related_name="uncon_experiments")
+    type = models.PositiveIntegerField(
+        null=False, blank=False, choices=ExperimentTypeChoices.choices, default=ExperimentTypeChoices.NEUROSCIENTIFIC
```

Comment:
same choices as the contrast experiment?

Reply:
same choices, but should be a different default choice